bundles/GraphicServiceOrder/Form/GsOrderLineForm.php

```diff
@@ -19,19 +19,19 @@ class GsOrderLineForm extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-      ->add('amount', TextType::class, [
+        ->add('amount', TextType::class, [
```

Comment:
Indentation (we may have to update `.php_cs.dist` to handle this automatically).

Reply:
Will be handled in coding standards PR.